install_config/configuring_sdn.adoc

```diff
@@ -316,15 +318,18 @@ flannel_interface=eth0
 using the `flannel_interface` variable. Without this variable, the
 {product-title} installation uses the default interface.
 
-NOTE: In order to provide custom networking CIDR for pods and services using
-flannel, there is a bugzilla being invested by Red Hat Bugzilla ID 1473858 at
-the time of writing this paper.
+[NOTE]
+====
+Custom networking CIDR for pods and services will be supported in a future release.
```

Comment:
Custom networking CIDR for pods and services **using flannel** will be supported in a future release.